packages/syft/src/syft/core/tensor/autodp/gamma_tensor.py

```diff
@@ -2381,6 +2382,188 @@ def _squeeze(state: dict) -> jax.numpy.DeviceArray:
             sources=output_state,
         )
 
+    def any(
+        self,
+        axis: Optional[Union[int, Tuple[int, ...]]] = None,
+        keepdims: Optional[bool] = False,
+        where: Optional[ArrayLike] = None,
+    ) -> GammaTensor:
+        func = "any"
+
+        def _any(state: dict) -> jax.numpy.DeviceArray:
+            return jnp.any(
+                *[
+                    i.reconstruct() if isinstance(i, GammaTensor) else i
+                    for i in state.values()
+                ]
+            )
+
+        output_state = dict()
+        output_state[self.id] = self
+
+        mapper[func] = _any
+
+        if where is None:
+            out_child = np.array(self.child.any(axis=axis, keepdims=keepdims))
+        else:
+            out_child = np.array(
+                self.child.any(axis=axis, keepdims=keepdims, where=where)
+            )
+
+        new_data_subjects = DataSubjectArray.from_objs(np.empty(out_child.shape))
+
+        return GammaTensor(
+            child=out_child,
+            data_subjects=new_data_subjects,
+            min_vals=lazyrepeatarray(data=False, shape=out_child.shape),
```

Comment:
It appears that we might be accidentally adding a new type of tensor in here, yes? Have we created a boolean tensor type? I suspect this will create type errors somewhere although python can do some smart casting). In any case, I suggest we instead set min_vals and max_vals to 0 and 1.

Reply:
(ditto for other methods)

Reply:
well, bool is a subclass of int so the only issue that might appear would be from using type() on the elements of the lazyrepeatarray, but I will change it to be consistent

Reply:
fixed